doc/src/site/sphinx/deployment/load_mojo_pipeline.rst

```diff
@@ -0,0 +1,167 @@
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: bash
+
+            ./bin/spark-shell --jars license.sig,jars/sparkling-water-assembly_2.11-SUBST_SW_VERSION-all.jar
+
+        .. code:: bash
+
+            ./bin/sparkling-shell --jars license.sig
+
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: bash
+
+            ./bin/pyspark --jars license.sig --py-files py/build/dist/h2o_pysparkling_SUBST_SPARK_MAJOR_VERSION-SUBST_SW_VERSION.zip
+
+        .. code:: bash
+
+            ./bin/pysparkling --jars license.sig
+
+
+At this point, you should have Spark interactive terminal where we can carry out predictions.
+For productionalizing the scoring process, we can use the same configuration,
+except instead of using Spark shell, we would submit the application using ``./bin/spark-submit``.
+
+Now Load the MOJO as:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.models.H2OMOJOPipelineModel
+            val settings = H2OMOJOSettings()
+            val mojo = H2OMOJOPipelineModel.createFromMojo("file:///path/to/the/pipeline_mojo.zip", settings)
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            from pysparkling.ml import H2OMOJOPipelineModel
+            settings = H2OMOJOSettings()
+            mojo = H2OMOJOPipelineModel.createFromMojo("file:///path/to/the/pipeline_mojo.zip", settings)
+
+In the examples above ``settings`` is an optional argument. If it's not specified, the default values are used.
+
+Prepare the dataset to score on:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            val dataFrame = spark.read.option("header", "true").csv("file:///path/to/the/data.csv")
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            dataFrame = spark.read.csv("file:///path/to/the/data.csv", header=True)
+
+And finally, score the mojo on the loaded dataset:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            val predictions = mojo.transform(dataFrame)
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            predictions = mojo.transform(dataFrame)
+
+We can select the predictions as:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        .. code:: scala
+
+            predictions.select("prediction")
+
+    .. tab-container:: Python
+        :title: Python
+
+        .. code:: python
+
+            predictions.select("prediction")
+
+The output data frame contains all the original columns plus the prediction column which is by default named
+``prediction``. The prediction columns contains all the prediction detail. Its name can be modified via the ``H2OMOJOSettings``
+object.
+
+Customizing the MOJO Settings
```

Comment:
Have you changed anything above this section?

Reply:
It's completely rewritten but the message remains the same